website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -158,22 +216,37 @@ We recommend keeping `critical` alerts non-dismissible as they are essential to
 </Hds::Alert>
 !!!
 
-## Placement
+## Multiple alerts
 
-### Page
+In general, displaying multiple alerts on the same page is discouraged as it can negatively impact the page hierarchy and unnecessarily shift emphasis away from the main content on the page.
 
-Page alerts are placed between the global header navigation and the breadcrumb, next to the left navigation.
+However, depending on the logic within the application and the type of content being expressed to the user, displaying more than one Alert on a page may be necessary.
 
-![Placement of page alert](/assets/components/alert/alert-placement-page.png =600x*)
+### Alerts of the same color
 
-### Inline
+While it might not always be possible depending on the type of content and intended action, try to refrain from displaying more than one of the same `color` Alert on the same page. This adds unnecessary visual weight to the page and may result in the user being confused about which Alert is more important or which might require attention first.
 
-Inline alerts can be added to a section or component or inline with content.
+![Example of multiple Alerts of the same color](/assets/components/alert/multiple-color-alerts-dont.png)
 
-![Placement of inline alert](/assets/components/alert/alert-placement-inline.png =600x*)
+To avoid this, try combining relevant or overlapping content into a single Alert with a more concise narrative, or by translating the related messages into a list.
 
-### Compact
+![Example of multiple Alerts combined into one](/assets/components/alert/multiple-color-alerts-do.png)
 
-Compact alerts can be added to a section or component or inline with content.
+![Multiple Alert messages combined in a bulleted list](/assets/components/alert/multiple-color-alerts-do-bulleted-list.png)
+
+### Alerts of the same type
+
+While not recommended for `page` type Alerts, it may be necessary to have more than one `inline` or `compact` Alert on a page. If this is the case, ensure that the Alerts are [ordered](#ordering-of-alerts) by impact and [placed](#placing-alerts) in areas where they provide the most relevance.
+
+### Alerts within a Reveal
+
+**As a last resort**, when multiple Alerts of the same type are necessary, display the Alerts that require immediate attention or have the most impact to the user within the page while obfuscating additional Alerts within a [Reveal](/components/reveal). We recommend against hiding `critical` and `warning` Alerts in this manner as they generally must be addressed quickly to avoid a negative outcome.
```

Comment:
I'm still unsure about this as a pattern recommendation. What do you think about adding a note and asking teams to include this in future product research and share the findings with us? Or some other way to help elicit more feedback around this pattern.

Reply:
I'm not opposed to removing it, I can add a note in a banner or something, but this it does seem helpful to have something like this as a "catch-all" or just-in-case.

If we're unsure about it (and think that it's unlikely to  happen frequently, if at all), then it might just be better to remove it entirely so that the idea doesn't exist out in the wild. What do you think?

Reply:
I think it's fine to keep it (this was ultimately the goal of the original ticket, after all), but I do think it's worth noting that we'd like real user feedback on this pattern from teams that do end up needing to use it.

Reply:
Added a bit of a note here requesting that consumers perform usability testing and share their findings with the team.